src/metadata/v57.json

```diff
@@ -1,4 +1,39 @@
 [
+  {
```

Comment:
**remarks:** those metadata are available since v55

They should also be added in v55 and v56

Reply:
All right. On it.